### PR TITLE
Add custom services.

### PR DESCRIPTION
**Pull request by Oberon00**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -387,6 +388,26 @@ public void onMessage(Message message) {
 }
 ```
 
+<a name="customservice"/>
+
+## Trace custom services
+
+You can use the SDK to trace custom services. A custom service is a meaningful part of your code that you want to trace but that does not fit any other tracer. An example could be the callback of a periodic timer.
```
Comment (SonjaChevre) [inline README.md]: From the official documentation, would be great to re-use here as well:

"A custom service is a service that has a manually defined entry point. You can use custom services to monitor service components that Dynatrace doesn't detect automatically."

Comment (SonjaChevre) [reply README.md]: hmmm good point

I took some time to re-think about it - and you are right, it's good as it is.

Comment (Oberon00) [reply README.md]: Mabe add: "Custom service tracers do not support tags. Therefore, if the service call is triggered by or triggers another service call (and this service call is not in the same process and thread), you should use a remote call tracer instead."

Comment (AlramLechnerDynatrace) [reply README.md]: other thread, but same process: > use in process linking
other process (and therefore other thread) > sync: remotecall, webrequest; async: messaging

Comment (Oberon00) [reply README.md]: Then change the last sentence to "Therefore, if the service call is triggered by or triggers another service call (and this service call is not in the same process and thread), you should use in-process linking (for in-process calls), remote call tracers (for cross-process/machine calls) or messaging (for asynchronous cross process/machine calls) instead."

I don't think we need to mention web requests here: if a webrequest was used, it should be clear from "that does not fit any other tracer" that a this is no use case for custom service tracers.

Comment (AlramLechnerDynatrace) [reply README.md]: You can use the SDK to trace custom service methods. A custom service mehod is a meaningful part of your code that you want to trace but that does not fit any other tracer. An example could be the callback of a periodic timer.

**Pull request merged**